[2021-06-25 09:20:09,942] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 2020-03-24T00:00:00+00:00 [queued]>
[2021-06-25 09:20:10,112] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 2020-03-24T00:00:00+00:00 [queued]>
[2021-06-25 09:20:10,113] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 09:20:10,113] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-06-25 09:20:10,113] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 09:20:10,168] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_3> on 2020-03-24T00:00:00+00:00
[2021-06-25 09:20:10,217] {standard_task_runner.py:52} INFO - Started process 9545 to run task
[2021-06-25 09:20:10,332] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_3', '2020-03-24T00:00:00+00:00', '--job-id', '346', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmptkweyxty', '--error-file', '/tmp/tmprlfe43sz']
[2021-06-25 09:20:10,343] {standard_task_runner.py:77} INFO - Job 346: Subtask task_3
[2021-06-25 09:20:11,175] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_3 2020-03-24T00:00:00+00:00 [running]> on host airflowvm
[2021-06-25 09:20:11,970] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2020-03-24T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-03-24T00:00:00+00:00
[2021-06-25 09:20:11,974] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-06-25 09:20:11,975] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-06-25 09:20:12,120] {subprocess.py:75} INFO - Output:
[2021-06-25 09:20:15,151] {subprocess.py:83} INFO - Command exited with return code 0
[2021-06-25 09:20:15,360] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_3, execution_date=20200324T000000, start_date=20210625T092009, end_date=20210625T092015
[2021-06-25 09:20:15,784] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-25 09:20:15,912] {local_task_job.py:151} INFO - Task exited with return code 0
